iotilebuild/iotile/build/config/site_scons/pythondist.py

```diff
@@ -154,10 +157,15 @@ def build_python_distribution(tile):
         target = env.Command(buildfile, inpath, Copy("$TARGET", "$SOURCE"))
         env.Depends(target, pkg_init)
 
+        if 'data' in os.path.normpath(buildfile).split(os.sep):
+            datafile = os.path.join(tile.support_distribution, outpath)
+            datafiles.append(datafile)
+
         buildfiles.append(buildfile)
 
     # Create setup.py file and then use that to build a python wheel and an sdist
     env['TILE'] = tile
+    env['datafiles'] = datafiles
```

Comment:
Since your `generate_setup.py` is building two files (`setup.py` and `MANIFEST.in`), you need to modify [line 174](https://github.com/iotile/coretools/pull/868/files#diff-7bf4f3fb3afa06b76d8a325626c25580R174) to include the MANIFEST.in file so that scons knows you're creating it.

You should also rename `generate_setup_py` to `generate_setup_and_manifest` so it's clear what it does.

Reply:
Fixed.

Reply:
Fixed.